server/views/pages/activities/startActivitiesMigration.njk

```diff
@@ -53,6 +53,20 @@
                             errorMessage: errors | findError('prisonId')
                         }) }}
 
+                        {{ govukInput({
+                            label: {
+                                html: 'When should the migrated activities start'
+                            },
+                            id: "activityStartDate",
+                            name: "activityStartDate",
+                            classes: "govuk-input--width-20",
+                            hint: {
+                                text: "Example 2025-02-05, must be earliest date of tomorrow."
+                            },
+                            value: form.activityStartDate,
+                            errorMessage: errors | findError('activityStartDate')
+                        }) }}
+
```

Comment:
Users can now enter a start date for when the migrated activities should become active